doc/overview.md

```diff
@@ -2,6 +2,23 @@
 
 <!---**ToDo**: Image/Map SCS context-->
 
+With this testbed, it is possible to run a full OSISM installation, the baseline of the Sovereign Cloud Stack, on an existing
+OpenStack environment such as Cleura or Open Telekom Cloud. If you are looking for the full installation, head over to here:
+<https://docs.osism.tech/>
+
+The testbed is intended as a playground. Further services and integration will be added over time. More and more best practices
+and experiences from the productive installations will be included here in the future. It will become more production-like over
+time. However, at no point does it claim to represent a production exactly.
+
+The section [How to contribute](/contributing.md) describes how to contribute to this testbed. Open Source Software lives from
+participation. We welcome any issues, change requests or general feedback. Do not hestiate to open an issue.
+
+For the German-speaking readers, [iX 2020/12](https://www.osism.tech/files/ix.2020.12.048_059.pdf_kiosk.pdf) contains an article
+by Kurt Garloff titled [Wolken-Verbund](https://www.osism.tech/files/ix.2020.12.048_059.pdf_kiosk.pdf), in which, among other
```

Comment:
The link doesn't work. I suggest to drop the complete sentence. Could be done in a followup though.